### PR TITLE
Ensure that monitoring operator has nodes to run

### PR DESCRIPTION
**Pull request by vrutkovs**
Fixes https://jira.coreos.com/browse/MON-268

### CODE REVIEW

File: roles/openshift_cluster_monitoring_operator/tasks/install.yaml [hunk 1]
```diff
@@ -1,4 +1,12 @@
 ---
+- name: Ensure that Monitoring Operator has nodes to run on
```
Comment (brancz) [inline roles/openshift_cluster_monitoring_operator/tasks/install.yaml]: It's probably good to be precise here and say the "Cluster Monitoring Operator"

File: roles/openshift_cluster_monitoring_operator/tasks/install.yaml [hunk 2]
```diff
@@ -1,4 +1,12 @@
...
+  import_role:
+    name: openshift_control_plane
+    tasks_from: ensure_nodes_matching_selector.yml
+  vars:
+    openshift_master_ensure_nodes_selector: "{{ openshift_cluster_monitoring_operator_node_selector | map_to_pairs }}"
+    openshift_master_ensure_nodes_service: Monitoring Operator
```
Comment (brancz) [inline roles/openshift_cluster_monitoring_operator/tasks/install.yaml]: Same as above.

**Comment by brancz** [comment]
This is not critical for 3.10, in 3.10 we only rolled this out to internal clusters, it's mostly important for 3.11.

File: roles/openshift_cluster_monitoring_operator/tasks/install.yaml (hunk 1 above)
Comment (vrutkovs) [reply roles/openshift_cluster_monitoring_operator/tasks/install.yaml]: Done

**Comment by brancz** [comment]
Thanks a lot for taking care of this!

/lgtm

**Pull request merged**